packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
+## [3.x.x]
```

Comment:
At the moment we don't have other features or fixes for this package so feel free to set an actual release version. 

The version should be added to: 
1. CHANGELOG.md
2. pubspec.yaml
3. packages/rx_bloc_cli/lib/src/rx_bloc_cli_constants.dart

Reply:
I dont think this small change deserves a new release. Let's just merge it and make it part of the next release when there are actual bug fixes or new features.

Reply:
It's small but important change in terms of DevX and security so I would prefer to proceed with a dedicated release instead of waiting for a bulk release.

Reply:
Done.